### PR TITLE
Added toggling sidebar.

### PR DESCRIPTION
**Pull request by sealionkat**
ESI-11136

### CODE REVIEW

**Comment by stepek** [comment]
screenshot-localhost 8080-2017-02-23-07-56-47 Please fix this

**Comment by stepek** [comment]
Could you add some css animation for this feature?

File: src/app/components/SidebarMenu.jsx [hunk 1]
```diff
@@ -58,5 +60,6 @@ export default class SidebarMenu extends Component {
 }
 
 SidebarMenu.propTypes = {
-  menuItems: PropTypes.array
+  menuItems: PropTypes.array,
+  openSidebar: PropTypes.bool
```
Comment (stepek) [inline src/app/components/SidebarMenu.jsx]: Please change name of prop ```openSidebar```.
 ```open``` will be  enough

File: src/app/components/SidebarMenu.jsx [hunk 2]
```diff
@@ -40,8 +40,10 @@ export default class SidebarMenu extends Component {
   }
 
   render() {
+    const sidebarOpenClass = this.props.openSidebar ? '' : 'sidebar-hidden';
+
     return (
-      <div className="pt-elevation-2 pt-fixed-top sidebar">
+      <div className={'pt-elevation-2 pt-fixed-top sidebar ' + sidebarOpenClass}>
```
Comment (stepek) [inline src/app/components/SidebarMenu.jsx]: Please use es6 templates. ``` `pt-elevation-2 pt-fixed-top sidebar ${sidebarOpenClass}` ```

File: css/sass/main.scss [hunk 3]
```diff
@@ -99,6 +102,10 @@ body {
   .sidebar-search {
     padding-left: 10px;
   }
+
+  &.sidebar-hidden {
+    display: none
```
Comment (stepek) [inline css/sass/main.scss]: semicolon

Comment (thealmightyone) [inline css/sass/main.scss]: also this class will prevent any animation being carried out.

Comment (sealionkat) [inline css/sass/main.scss]: Yeah, I know. I'll change it.

**Comment by sealionkat** [comment]
CSS transitions added.

File: src/app/App.jsx [hunk 4]
```diff
@@ -63,9 +60,21 @@ class App extends Component {
...
+      });
+    } else {
+      this.setState({
+        openSidebar: true,
+        contentClassNames: 'view-content'
+      });
+    }
   };
+
+
```
Comment (stepek) [inline src/app/App.jsx]: please remove one empty line

**Pull request merged**